app/views/authors/_form.html.erb

```diff
@@ -11,27 +11,27 @@
 <div class="w-full max-w-xs">
   <%= form_with(model: @author, local:true) do |f| %>
     <div class="mb-4">
-      <%= f.label :first_name, class: "block text-gray-700 text-sm font-bold mb-2" %><br>
+      <%= f.label t('authors.fname'), class: "block text-gray-700 text-sm font-bold mb-2" %><br>
```

Comment:
read this:
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

add translations for attributes for Author model

Reply:
```
en:
  activerecord:
    attributes:
      author:
        first_name: First name
```